src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.tsx

```diff
@@ -196,11 +186,11 @@ function FloatingActionButtonAndPopover(props) {
                         text: props.translate('sidebarScreen.saveTheWorld'),
                         onSelected: () => interceptAnonymousUser(() => Navigation.navigate(ROUTES.TEACHERS_UNITE)),
                     },
-                    ...(!props.isLoading && !Policy.hasActiveFreePolicy(props.allPolicies)
+                    ...(!props.isLoading && !Policy.hasActiveFreePolicy(props.allPolicies ?? [])
                         ? [
                               {
                                   displayInDefaultIconColor: true,
-                                  contentFit: 'contain',
+                                  contentFit: 'contain' as ImageContentFit,
```

Comment:
```suggestion
                                  contentFit: 'contain' as const,
```
I think this will work